app/controllers/games_controller.rb

```diff
@@ -3,11 +3,12 @@ class GamesController < ApplicationController
 
   def index
     @games = Game.all
```

Comment:
This currently returns all games whether or not they can be joined but since I am tasked with making the ability to join a game I can work this into that story. Just mentioning this because I may change it back to Game.avail when making the join feature unless there's a reason you think I shouldn't